Libplanet.Tools/bin/planet

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+"use strict";
+const child_process = require("child_process");
+const fs = require("fs");
+const path = require("path");
+const { BIN_PATH, BIN_NAME, download } = require("../download.js");
+
+const exePath = path.join(BIN_PATH, BIN_NAME);
+
+function runExe() {
+  child_process.spawn(exePath, process.argv.slice(2), { stdio: "inherit" })
+    .on("exit", process.exit);
+}
+
+fs.stat(exePath, (err, stats) => {
+  if (!err && stats.isFile() && stats.size > 64 * 1024) return runExe();
```

Comment:
Do we need to check the file's size?

Reply:
This heuristics purposes to determine if the file is a node.js script file (< 64 KiB) or an actual executable binary (> 64 Kib), but indeed it's hardly understandable its intention.  I am going to adjust this code so that it's more easily understandable.